crates/services/upgradable-executor/src/lib.rs

```diff
@@ -3,6 +3,13 @@
 #![deny(unused_crate_dependencies)]
 #![deny(warnings)]
 
+#[cfg(test)]
+mod _ignore {
+    #![allow(unexpected_cfgs)] // for cfg(coverage)
+    #[cfg(coverage)]
+    use ntest as _; // Only used outside cdg(coverage)
```

Comment:
You can move it to `executor.rs`
<img width="486" alt="image" src="https://github.com/FuelLabs/fuel-core/assets/18346821/21c12b9c-9b14-4906-a2ac-41eda33a6ccf">

Reply:
Sure, done in 1c6f607e79dd0c8dbbee8c8610bb949c0bd45b6b